packages/side-nav/src/SideNavItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { MenuItem, MenuItemProps } from '@leafygreen-ui/menu';
+import { LEFT_RIGHT_OFFSET } from './styles';
+
+// style overrides for menu item
+const containerStyleOverrides = css`
+  padding: 10px ${LEFT_RIGHT_OFFSET}px 10px ${LEFT_RIGHT_OFFSET}px;
+  min-height: 0;
+  border-radius: 5px;
+
+  &:before {
+    content: normal;
+  }
+
+  &:focus:active {
+    background-color: ${uiColors.gray.light2};
+  }
+`;
+
+const titleTextStyleOverrides = css`
+  font-size: 15px;
+  text-transform: capitalize;
+`;
+
+const disabledStyleOverrides = css`
+  background-color: ${uiColors.white};
```

Comment:
We should probably set to transparent (assuming the intent is for it to not have a visible background). Additionally, we probably should change the color to a lighter gray to make disabled states more apparent.

Reply:
The lighter color is already taken care of by the default styles for `MenuItem`, but I've updated the background.